CHANGELOG.md

```diff
@@ -38,10 +40,17 @@
 
 ### Changed
 
+- [Add required_ruby_version >= 2.7.0 #125](https://github.com/kufu/activerecord-bitemporal/pull/125)
+- [Update auto asgn #124](https://github.com/kufu/activerecord-bitemporal/pull/124)
+- [Update License and CoC files #115](https://github.com/kufu/activerecord-bitemporal/pull/115)
+
 ### Deprecated
 
 ### Removed
 
+- [Remove Gemfile.lock #126](https://github.com/kufu/activerecord-bitemporal/pull/126)
+- [Drop support Rails 5.2 #122](https://github.com/kufu/activerecord-bitemporal/pull/122)
```

Comment:
In general, supported version changes (#122 #125) are classified as breaking changes.

Reply:
Thanks, I modified it.